en-US/firefox/whatsnew_77.lang

```diff
@@ -0,0 +1,53 @@
+## NOTE: Demo page at https://www-dev.allizom.org/firefox/77.0/whatsnew/all/
```

Comment:
Double check the link, the demo link is not set up yet. Or should we use this: https://www-demo3.allizom.org/en-US/firefox/77.0/whatsnew/all/?

Reply:
it's not on dev yet because the PR hasn't merged, but we can point to demo3 instead if we want to get strings out faster.